monkestation/code/modules/antagonists/living_lube/living_lube_abilities.dm

```diff
@@ -0,0 +1,40 @@
+/obj/effect/proc_holder/spell/aoe_turf/knock/living_lube
+	action_background_icon_state = "bg_hive" //closest to a pink spell color we have
+	invocation = "AULIE HONKSIN FIERA"
+	invocation_type = "whisper"
+
+/obj/effect/proc_holder/spell/aimed/banana_peel/living_lube
+	action_background_icon_state = "bg_hive"
+
+/obj/effect/proc_holder/spell/voice_of_god/clown/living_lube
+	action_background_icon_state = "bg_hive"
+	power_mod = 0.5 //Slightly more annoying
+
+/obj/effect/proc_holder/spell/targeted/smoke/living_lube
+	action_background_icon_state = "bg_hive"
+
+/obj/effect/proc_holder/spell/targeted/displacement
+	name = "Displacement"
+	desc = "Force someone through the clown dimension and launch them out somewhere else on the station."
+	charge_max = 90 SECONDS
+	clothes_req = FALSE
+	range = 7
+	invocation_type = "none"
+	include_user = TRUE
+	action_icon = 'icons/mob/actions/actions_spells.dmi'
+	action_icon_state = "blink"
+	action_background_icon_state = "bg_hive"
+
+/obj/effect/proc_holder/spell/targeted/displacement/cast(list/targets, mob/user = usr)
+	var/target = targets[1]
+
+	if(!isliving(target))
+		return
+
+	if(get_dist(user,target)>range)
+		to_chat(user, "<span class='notice'>\The [target] is too far away!</span>")
+		return
+
+	do_teleport(target,find_safe_turf(),asoundin = 'sound/items/bikehorn.ogg')
```

Comment:
Note that this can teleport them ANYWHERE on the station.
Including the AI upload/sat, armoury and possibly even inside the engine.

Reply:
> inside the engine.

PRANKED